projects/storefrontstyles/scss/components/misc/chat-messaging/_messaging.scss

```diff
@@ -0,0 +1,139 @@
+%cx-messaging {
+  .container {
+    display: flex;
+    flex-direction: row;
+    padding: 15px 0;
+    min-height: 400px;
+    padding-bottom: 2rem;
+
+    .cx-avatar-line {
+      border: 0px solid;
+      margin: 0 0 0 2.1875rem;
+      color: #d3d6db;
+      border-right-width: 1px;
+    }
+
+    .cx-messages {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      gap: 1rem;
+      margin: 0 0 0 -2.1875rem;
+      max-height: 400px;
+      overflow-y: auto;
+
+      label {
+        margin-bottom: 0;
+        font-size: var(--cx-font-size, 0.875rem);
+      }
+
+      .cx-message-card {
+        display: flex;
+        flex-direction: row;
+        gap: 0.875rem;
+
+        div {
+          width: 100%;
+        }
+      }
+
+      .cx-message-display {
+        @include media-breakpoint-up(md) {
+          padding: 0 0 0 4.875rem;
```

Comment:
This should be `padding-top: 0; padding-inline-end: 0; padding-bottom: 0; padding-inline-start: 4.875rem`
any padding or margin that has left or right values should be `padding-inline-start:`, `padding-inline-end` or `margin-inline-end` or `margin-inline-start`. We do this to make our padding and margin work with directionality for languages that read right to left or left to right.

Reply:
Ohh, I see.
`padding-inline-start: 4.875rem` is sufficient in that case.
I added it this was because `padding-left` was failing pipeline.